herbert_core/classes/MPIFramework/@ClusterDummy/ClusterDummy.m

```diff
@@ -0,0 +1,142 @@
+classdef ClusterDummy < ClusterWrapper
+    % The class to support Dummy cluster
+    %
+    %----------------------------------------------------------------------
+
+    properties(Access=protected, Hidden=true)
+        last_results_;
+    end
+
+    methods
+        function obj = ClusterDummy(n_workers,mess_exchange_framework,log_level)
+            % Constructor, which initiates wrapper around Dummy
+            % MPI framework.
+            %
+            % The wrapper provides common interface to run various kind of
+            % Herbert parallel jobs.
+            %
+            % Empty constructor generates wrapper, which has to be
+            % initiated by init method.
+            %
+            % Non-empty constructor calls the init method itself
```

Comment:
If nargin==1, n_workers is consumed but not passed to init. Comment may need rewording, or nargin condition below.

Reply:
This is *not* MPI, this is a dummy MPI framework so that you can call MPI routines in serial on the local process to allow you to profile it and quickly check whether your job executor is valid and working.

This is why it only allows one process and lots of the routines are empty/dummied out. They aren't needed if we're not really parallel, but have to conform to the structure laid out by the abstract MPI framework classes.

Reply:
My comment was about the discrepancy between the comments and the code.

As we are here though, your reply looks like a good comment to put at the head of the class